Simple-Multiparty/README.md

```diff
@@ -4,50 +4,24 @@ Note: Read the README.md file in the Basic-Video-Chat folder before starting her
 
 ## Subcribing to multiple streams
 
-Previous samples subscribe to only one stream. In a multiparty video audio call
-there should be multiple parties.
+Previous samples subscribed to only one stream. In a multiparty video audio call
+there are multiple streams.
 
 ```java
-    @Override
-    public void onStreamReceived(Session session, Stream stream) {
-        Log.d(TAG, "onStreamReceived: New stream " + stream.getStreamId() + " in session " + session.getSessionId());
-
-        if (mSubscribers.size() + 1 > MAX_NUM_SUBSCRIBERS) {
-            Toast.makeText(this, "New subscriber ignored. MAX_NUM_SUBSCRIBERS limit reached.", Toast.LENGTH_LONG).show();
-            return;
-        }
-
-        final Subscriber subscriber = new Subscriber(MainActivity.this, stream);
-        mSession.subscribe(subscriber);
-        mSubscribers.add(subscriber);
-        mSubscriberStreams.put(stream, subscriber);
-
-        int position = mSubscribers.size() - 1;
-        int id = getResources().getIdentifier("subscriberview" + (new Integer(position)).toString(), "id", MainActivity.this.getPackageName());
-        RelativeLayout subscriberViewContainer = (RelativeLayout) findViewById(id);
-
-        subscriber.setStyle(BaseVideoRenderer.STYLE_VIDEO_SCALE, BaseVideoRenderer.STYLE_VIDEO_FILL);
-        subscriberViewContainer.addView(subscriber.getView());
-
-        id = getResources().getIdentifier("toggleAudioSubscriber" + (new Integer(position)).toString(), "id", MainActivity.this.getPackageName());
-        final ToggleButton toggleAudio = (ToggleButton) findViewById(id);
-        toggleAudio.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
-            public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
-                if (isChecked) {
-                    subscriber.setSubscribeToAudio(true);
-                } else {
-                    subscriber.setSubscribeToAudio(false);
-                }
-            }
-        });
-        toggleAudio.setVisibility(View.VISIBLE);
-    }
+@Override
+public void onStreamReceived(Session session, Stream stream) {
+    Log.d(TAG, "onStreamReceived: New stream " + stream.getStreamId() + " in session " + session.getSessionId());
+
+    final Subscriber subscriber = new Subscriber.Builder(MainActivity.this, stream).build();
+    mSession.subscribe(subscriber);
+    addSubscriber(subscriber);
+}
```

Comment:
Should we detail what `addSubscriber` does since the view is actually updated there now?

Reply:
I'm leaning towards no.  Reason being, the previous code, which was lengthier, didn't add any details describing that each portion of the code did.  (The preceding passage seemed sufficient).

That said, what did you have in mind?

Reply:
Sounds good - just wanted to get an opinion.